src/scripts/load.py

```diff
@@ -0,0 +1,75 @@
+"""
+Script to be run to insert existing data from a spreadsheet to the database.
+"""
+
+import csv
+from affiliations.models import Affiliation, Submitter, Coordinator
+
+
+def run():  # pylint: disable-msg=too-many-locals
+    """Iterate through CSV and create Affiliation, Submitter ID, and Coordinator
+    objects in the DB."""
+
+    with open(
+        "/Users/gabriellasanchez/Desktop/repos/stanford-affils/src/scripts/test_affils.csv",
+        encoding="utf-8",
+    ) as file:
+        csv_reader = csv.DictReader(file)
+
+        Affiliation.objects.all().delete()
```

Comment:
If this deletes all existing affiliations in the DB, it would be good to mention that in the docstring for this script.

Reply:
In test, this this helpful to have a clean slate, however in live when we actually migrate this data this will not be necessary as we will be starting from a new instance. 

Should this be removed for this instance? or kept in for the test use case?

Reply:
IMO, if we're going to leave a script in version control that bills itself as a script that imports CSV into the database, we should either be very careful to point out to future maintainers that if you run this script, it will blow away existing affiliations, or we should remove this script from version control after we're done with it.

Reply:
I think then this specific line can be removed, and the script can be for importing CSV into the database to make it clear what the intended use is. Also, since when we run this script, this line will be unnecessary.